components/library_screen/Library_Hours.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect} from 'react';
+import { StyleSheet, View, Text, Linking, FlatList, ScrollView } from "react-native";
+
+import { Card, Title, Subheading, Paragraph, Button } from 'react-native-paper';
+
+import {db} from '../../firebase/config';
+import { doc, getDoc } from 'firebase/firestore';
+
+// set Locations
+const latitude = "38.34882";
+const longitude = "-97.201";
+const label = "Tabor College Library, 400 S Jefferson St, Hillsboro, KS 67063";
+
+// this detect if Phones installed Maps and open Automatically 
+const url = Platform.select({
+  ios: "maps:" + latitude + "," + longitude + "?q=" + label,
+  android: "geo:" + latitude + "," + longitude + "?q=" + label
+});
+
+const LibraryHours = () => {
+    const [libraryData, setLibraryData] = useState([]);
+
+    useEffect(() => {
+        // get Jay Shop data from Firestore
+        const docRef = doc(db, "helpful hours example", "library hours");
+        getDoc(docRef)
+            .then((doc) => {
+                const data = doc.data();
+                setLibraryData(data);
+            })
+            .catch((err) => {
+                console.log(err);
+            })
+    }, []);
+
+    // Deconstruct Just to call the open hours
+    const {"Contact information":contactInfo,"open hours": openHours } = 
+        libraryData;
+
+    return (
+        <Card style = {styles.card}>
+            <Title style ={styles.title}>{libraryData.title}</Title> 
+                <Card.Cover style = {styles.coverImage} source = {{uri: libraryData.image}}/>
+                <Paragraph style = {styles.memo}>{libraryData.memo}</Paragraph>
+                    <FlatList 
+
+                        showsVerticalScrollIndicator={false}
+                        listKey={(_, index) => `_key${index}`}
+                        keyExtractor={(_, index) => `_key${index}`}
+                        data={openHours}
+                        renderItem={({ item }) => (
+                            <View style = {styles.alignSideBySide}>
+                            {/* Show days */}
+                                 <Subheading styles ={styles.days} >{item.days}</Subheading>
+                                    {/* Map out the hours */}
+                                    <FlatList
```

Comment:
Let's not use FlatList to map out the hours. You can use the JavaScript Map method to not make the section scrollable.

Reply:
how to implement using JS map ?

Reply:
It would just be openHours.map((item) => (
.
.
.
)

It's simpler than FlatList.

Reply:
Actually, don't worry about it. I can go ahead and apply the changes myself. You can make Library.jsx scrollable.

Reply:
okay will implement when i get home on Friday or before.

Reply:
okay i will make it scrollable